packages/consensus/src/validation/block.cairo

```diff
@@ -69,6 +68,9 @@ pub fn compute_and_validate_tx_data(
         txids.append(txid);
 
         if (is_coinbase) {
+            if (is_coinbase_txid_duplicated(txid, block_height)) {
```

Comment:
Please add a comment that we do not add duplicate UTXO to the accumulator hence it's not possible to spend this coinbase tx